tests/e2e/utils/track-event.js

```diff
@@ -10,14 +10,27 @@
  *
  * @param {Page} page
  * @param {string} eventName Event name to match.
+ * @param {string|null} URLpath The URL path where the event should be triggered.
  * @return {Promise<Request>} Matching request.
  */
-export function trackGtagEvent( page, eventName ) {
+export function trackGtagEvent( page, eventName, URLpath = null ) {
```

Comment:
Not sure what the recommended format is for naming, but wouldn't `urlPath` or `URLPath` be the better choice?